plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/webhooks/WebhookUtil.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors: Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.internal.ui.webhooks;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import com.openshift.restclient.model.IBuildConfig;
+import com.openshift.restclient.model.build.BuildTriggerType;
+import com.openshift.restclient.model.build.IBuildTrigger;
+import com.openshift.restclient.model.build.IWebhookTrigger;
+
+public class WebhookUtil {
```

Comment:
@adietish  advised me earlier that we need to add author comment headers to classes.